src/SpotifySongSearch.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { getToken } from "./utils/tokenStorage";
+import { fetchSpotifyTracksIds } from "./utils/spotify";
+
+const parseArtistsTracks = (text) => {
+  return text.split("\n").map((artistTrackStr) => {
+    const splitArtistTrack = artistTrackStr.split(" - ");
+
+    if (splitArtistTrack.length < 2)
+      return { artist: splitArtistTrack[0].trim(), track: "" };
+
+    return {
+      artist: splitArtistTrack[0].trim(),
+      track: splitArtistTrack[1].trim(),
+    };
+  });
+};
+
+const TrackView = ({ track, key }) => {
```

Comment:
**key** is a special react attribute - it exist on any react component. so you should never pass it via props. Use it like <TrackView key={mykey}